problem2.R

```diff
@@ -0,0 +1,58 @@
+# Problem 2 - Git and Github Assignment
+
+# Load required packages ---
+library(tidyverse)
+# ---
+
+# Load raw data ---
+raw_data <- readLines("suites_dw_Table1.txt")
+# ---
+
+# Saving the variable descriptions ---
+
+# Locate the separation row between variable descriptions and the values
+L <- which(raw_data == " ")
+
+# Save variable descriptions in a variable
+var_descr <- raw_data[1:(L-1)]
+
+# Save the variable descriptions in a text file
+cat(
+  var_descr,
+  file = "variable_description.txt",
+  sep = ";"
```

Comment:
The descriptions are already separated by ";". This results in a double separating semicolon ";;"